docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
     environment:
       - DATABASE_URL=postgresql://postgres:postgres@db/postgres
       - DATABASE_DISABLE_SSL=true
-      - CELERY_BROKER_URL=redis://redis
```

Comment:
why the change? I thought the generical naming was better, if in the future the broker is channged to RabbitMQ... the name of the env won't have to change in the other references. Anyway it is ok to keep going.

Reply:
When Heroku provisions Redis, it sets up a `rediss://` URL and I could only make it work using that protocol.